diff --git a/node_modules/uri-template-router/index.js b/node_modules/uri-template-router/index.js
index 95262af..40351de 100644
--- a/node_modules/uri-template-router/index.js
+++ b/node_modules/uri-template-router/index.js
@@ -275,7 +275,7 @@ Route.prototype.decode = function decode(uri){
 						result[varname] = value.split(segment.separator).map(decodeURIComponent);
 					}
 				}else if(value){
-					result[varname] = decodeURI(value);
+					result[varname] = decodeURIComponent(value);
 				}
 			}
 		}
